src/Stratis.FederatedPeg.IntegrationTests/Utils/PowerShellScriptGeneratorAsTests.cs

```diff
@@ -44,19 +27,8 @@ public PowerShellScriptGeneratorAsTests(ITestOutputHelper output)
         [Fact]
         public void Generate_PS1_Fragment()
         {
-            this.mainchainNetwork = Networks.Stratis.Regtest();
-            this.sidechainNetwork = (FederatedPegRegTest)FederatedPegNetwork.NetworksSelector.Regtest();
-
-            this.mnemonics = this.sidechainNetwork.FederationMnemonics;
-            this.pubKeysByMnemonic = this.mnemonics.ToDictionary(m => m, m => m.DeriveExtKey().PrivateKey.PubKey);
-
-            this.scriptAndAddresses = GenerateScriptAndAddresses(this.mainchainNetwork, this.sidechainNetwork, 2, this.pubKeysByMnemonic);
-
-            this.federationMemberIndexes = Enumerable.Range(0, this.pubKeysByMnemonic.Count).ToList();
-            this.chains = new[] { "mainchain", "sidechain" }.ToList();
-
             var stringBuilder = new StringBuilder();
-            this.newLine = s => stringBuilder.AppendLine(s);
+            this.newLine = s => stringBuilder.AppendLine();
```

Comment:
are you sure ? sorry I can't see how it works :/

Reply:
Pardon me.  I've included `s`.  Cheers.